### PR TITLE
update presets

### PR DESCRIPTION
**Pull request by goosewobbler**
* Add infuraSepolia
* Remove ws endpoint for Optimism (doesn't work, been removed from Frame for a while)
* Remove idChain
* Remove space from gnosis

### CODE REVIEW

File: presets/index.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ module.exports = (options = {}) => {
     alchemyGoerli: [`wss://eth-goerli.ws.alchemyapi.io/v2/${options.alchemyId}`, `https://eth-goerli.alchemyapi.io/v2/${options.alchemyId}`],
     infuraPolygon: [`wss://polygon-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://polygon-mainnet.infura.io/v3/${options.infuraId}`],
     infuraArbitrum: [`wss://arbitrum-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://arbitrum-mainnet.infura.io/v3/${options.infuraId}`],
-    infuraOptimism: [`wss://optimism-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://optimism-mainnet.infura.io/v3/${options.infuraId}`],
-    idChain: ['wss://idchain.one/ws/'],
-    gnosis: ['https://rpc.gnosischain.com '],
+    infuraOptimism: [`https://optimism-mainnet.infura.io/v3/${options.infuraId}`],
```
Comment (mholtzman) [inline presets/index.js:18]: does Infura no longer have have a ws endpoint for Optimism?

Comment (goosewobbler) [reply presets/index.js:18]: No it doesn't work - this has been https only in Frame for a while:

https://github.com/floating/frame/pull/453

**Comment by mholtzman** [comment]
@goosewobbler aren't there endpoint tests for some of these that are broken now?

**Comment by goosewobbler** [comment]
@mholtzman possibly - checking.  Additionally, we need the tests that we can run on CI to be run on CI

**Comment by goosewobbler** [comment]
@mholtzman they all pass but I need to add one for Sepolia

**Comment by mholtzman** [comment]
> @mholtzman they all pass but I need to add one for Sepolia

how do the tests for the idchain endpoint pass?

**Comment by goosewobbler** [comment]
oh I didn't update my local, nvm.  I'm updating them anyway

**Review by mholtzman — APPROVED** [review]

**Pull request merged**